plugins/UsersManager/Validators/Email.php

```diff
@@ -47,7 +47,7 @@ public function validate($value)
     private function isUnique($email)
     {
         if (APIUsersManager::getInstance()->userExists($email)) {
-            throw new Exception(Piwik::translate('UsersManager_ExceptionLoginExists', $email));
+            throw new Exception(Piwik::translate('UsersManager_ExceptionLoginExistsAsEmail', $email));
```

Comment:
As we are checking the email if it already exists I think the correct error message would be `UsersManager_ExceptionEmailExistsAsLogin`

And the error below should be `UsersManager_ExceptionEmailExists`